distribution/docs/src/main/resources/_contents/_core-concepts/registry-intro-contents.adoc

```diff
@@ -0,0 +1,4 @@
+
+== Registry
+
+${ddf-registry} is a
```

Comment:
@mcalcote I'm going to go ahead and add additional details to this section and make it read a little more correct.